### ISSUE TITLE
Games Takes Very Long to Load Up

### ISSUE THREAD

**Issue by ekianjo** [virtual-programming/specops-linux#12]
I had no issue whatsoever at the beginning, the game used to load up quite fast and I would reach the early screens with the logos of the different companies involved in making the game. But recently, as I progressed in the last part of the game, somehow there is a hiatus and it takes make 2 minutes before the game starts properly - for no particular reason. Is there any diagnostic I can run to find out what's happening ?

**Comment by robertc** [comment]
Yes, we know: https://github.com/virtual-programming/specops-linux/issues/5

**Comment by ekianjo** [comment]
It's actually different I think. I have very good frame rates in game and in the menu. It's just that between the time I click on "play" in Steam and the time something happens, there's 2 minutes in between. It used to be almost instantaneous.

**Comment by licaon-kter** [comment]
Might be that issue anyway, read from my post down: https://github.com/virtual-programming/specops-linux/issues/5#issuecomment-107149219
